src/test/java/com/mmm/his/cer/utility/farser/ast_complex/AstLimitationsTest.java

```diff
@@ -64,9 +64,22 @@ public void doesNotProperlyWork_OperatorWithParameters() throws Exception {
     assertThat(lines, is(new String[] {
         "IN",
         "  a",
-        "  1," // ... stuff is missing here
+        "  1" // ... stuff is missing here
```

Comment:
This still won't work because it is a non-terminal operator. I only implemented the function args for terminal nodes.

Reply:
Ah, so `a & IN(1, 2, 3)` would work?

Reply:
That would still fail because `&` and `IN` are both operators, and two operators can't be next to each other. It would have to be something like `a IN someFunctionName(1, 2, 3)`, so the left returns `a` and right returns the result of `someFunctionName(1, 2, 3)`, and then IN operates on that and returns the result

Reply:
Aaah I see, ok

Reply:
correct